src/lib/pages/frontsite/Documents/DocumentDetails/DocumentDetails.js

```diff
@@ -126,8 +126,10 @@ DocumentDetailsLayout.defaultProps = {
 
 class DocumentDetails extends Component {
   componentDidMount() {
-    const { match } = this.props;
-    this.fetchDocumentsDetails(match.params.documentPid);
+    const { documentDetails, match } = this.props;
+    if (Object.keys(documentDetails.metadata).length === 0) {
```

Comment:
it would be much easier to just export (out of this library) the version of this component without redux mapping to be able to to use it outside, none of these changes would be necessary then.

Reply:
I still need to keep some of the changes otherwise it is not working.